src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int NAME_POSITION = 0;
+    static final int NUM_POSITION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        int sumSupply = 0;
+        int sumBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+
+        String[] array = builder.toString().split(System.lineSeparator());
+
+        for (String arr : array) {
```

Comment:
You should check if `arr` is empty before splitting it. This will prevent potential `ArrayIndexOutOfBoundsException` if there are empty lines in the input file.